### ISSUE TITLE
Verification needed

### ISSUE THREAD

**Issue by nomisunrider** [sniffingpickles/Auto-Amazon-Giveaways#12]
After it logs in to Amazon, I see it gets a prompt from Amazon to email me a verification code that needs to be verified.  To get around this, can an option be added to use a different browser profile directory?

**Comment by sniffingpickles** [comment]
turn off all 2fa stuff

**Comment by dziban303** [comment]
This also happens if you try to run the bot while on a VPN. If you use a
VPN, turn it off when running the bot, or set up an exception.

**Comment by nomisunrider** [comment]
I do not have two-factor enabled and am not on a VPN.
It states it is because I am signing in from a new device

![image](https://user-images.githubusercontent.com/13275125/43542529-e1b3eaf4-9592-11e8-9e62-6a517af65d34.png)

**Comment by sniffingpickles** [comment]
If you're on windows, right click the chromium task on the start menu, and click the Chromium application.  That way it opens a new unattached browser session of chromium.  Log in and verify your stuff.  If that doesn't work, double check you don't have any alert settings here:

https://www.amazon.com/a/settings/approval

**Comment by nomisunrider** [comment]
I'm on linux.  I think the issue is that when python launches chromium, it does not have any browsing history or cookies for amazon, so detects it as a new device.

Regarding that link, I have nothing setup for 2FA.  I see this:

![image](https://user-images.githubusercontent.com/13275125/43545389-5d1b7632-959b-11e8-801e-ca16767aa12f.png)

**Comment by sniffingpickles** [comment]
My best advice is to try and figure out how to log in via chromium at least once, as mentioned before.  Then amazon may think you're just logging in normally.

**Comment by nomisunrider** [comment]
I figured out a way around it.  
As suggested, I had to login once with a verification code, but there is no way to launch the pypeteer chromium easily on linux (that i can find).  I threw the below together and was able to login once, so now it no longer gets the verification prompt.  It must have stored a cookie "somewhere."


`import asyncio
from pyppeteer import launch

async def main():
    browser = await launch({'headless': False})
    page = await browser.newPage()
    await page.goto('http://amazon.com')
    await page.waitFor(300000);
asyncio.get_event_loop().run_until_complete(main())
`
